docs/core/deploying/trimming/trim-warnings/il2092.md

```diff
@@ -27,3 +27,7 @@ public class Derived : Base
   public override void TestMethod([DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicFields)] Type type) {}
 }
 ```
+
+## Fixing
+
+See [Fixing Warnings](../fixing-warnings.md#dynamicallyaccessedmembers) for guidance.
```

Comment:
I'm not sure how much this documentation would help with the override case, the problem here is not that we dont have annotations, but that the annotations dont match. Likely prompting to combine the annotations and have according to the example `[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicMethods|DynamicallyAccessedMemberTypes.PublicFields)]`
So probably we need something to mention how to fix those scenarios, or remove the guidance from IL2092 to IL2095?

Reply:
Good point, I think the fix is pretty clear from the warning, so I'll remove that section on IL2092-IL2095.